hdf5-sys/src/h5es.rs

```diff
@@ -0,0 +1,84 @@
+//! Event set module
+use crate::internal_prelude::*;
+
+#[repr(C)]
+pub struct H5ES_err_info_t {
```

Comment:
Add common derives (like in other modules) for all these structs?

Reply:
Added derives